### ISSUE TITLE
Forward Old LX Email Addresses

### ISSUE THREAD

**Issue by lin-d-hop** [SolidarityEconomyAssociation/land-explorer-front-end#64]
LX had two emails set up:
hello@landexplorer.cc
admin@landexplorer.cc

With the shift of domains these have now stopped working. 

In the short term it would be good to use Sendgrid to forward any emails to these addresses to a Digital Commons address - landexplorer@digitalcommons.coop

We''ll need to add the MX details to the landexplorer.cc domain in domains.coop.
I'd planned to get this done before hols but the domain transfer took much longer and was much more complicated than expected so in the end, here we are. Sorry to drop this on the pile.

In the medium term we should figure out our strategy on all our apps and emails. But that is out of scope on this issue.

**Comment by King-Mob** [comment]
2FA on sendgrid stymied me before I could try anything out

**Comment by lin-d-hop** [comment]
Okay looking into this... neither Domains.coop or Sendgrid will allow simple forwarding. I've set up the domains so that sendgrid can serve as SMTP.... so my next step attempt will be to see if we can use the DigitalCommons.coop emails to also receive the landexplorer.cc emails.

Does anyone know who the host of our emails is? That info wasn't added to bitwarden (just username and password) so I don't know where to log in to access the master email account. Ping @ColmMassey @wu-lee @King-Mob 

If this fails then I think we just sunset those emails and trust that anything important will be resent to our new emails.

**Comment by lin-d-hop** [comment]
Using a tool called https://app.improvmx.com/ to set up free forwarding from those old accounts.
Details in Bitwarden.

Currently just waiting on domains.coop as am getting errors adding new MX records to the DNS.

**Comment by lin-d-hop** [comment]
Alright they fixed that one quick!

Tested and this is all set up and working. Huzzah.
